apps/www/components/admin/orderForm/OrderForm.tsx

```diff
@@ -64,9 +65,9 @@ export const OrderForm = React.memo<OrderFormProps>(({ status, orderId }) => {
   const handleSubmit = React.useCallback(
     async (body: OrderRequestBody) => {
       try {
-        await mutate(body);
+        return await mutate(body);
```

Comment:
```suggestion
        await mutate(body);
```
Ta funkcja nie może nic zwracać jeśli nie było błędu.

Reply:
oki